docs/sql/kill.txt

```diff
@@ -0,0 +1,36 @@
+.. highlight:: psql
+.. _kill_job:
+
+=========
+Kill jobs
+=========
+
+Each request in Crate that fetches or manipulates data is called "job".
+Jobs that are currently executed are logged in the system table ``sys.jobs`` 
+(see :ref:`jobs_operations_logs`).
+
+Killing an active job forces Crate to stop its execution on the cluster 
+immediately. There are two different SQL commands available for killing jobs.
+
+The ``KILL ALL`` statement stops every single job on each node that is 
+currently running. It returns the number of job-contexts that have been killed 
+per node.
+
+::
+
+    cr> kill all;
+    KILL OK, ... rows affected (... sec)
+
+
+
+Using ``KILL job_id`` only the specified ``job_id`` will be stopped on every 
```

Comment:
`KILL job_id` only kills one single job with the specified job id. Like `KILL ALL` it returns the total number of contexts of that job killed on all nodes.